libvast/CMakeLists.txt

```diff
@@ -139,6 +139,15 @@ endif ()
 
 # -- caf -----------------------------------------------------------------------
 
+function (systemize_target tgt)
```

Comment:
Can you rename this to `VASTSystemizeTarget` so it is guaranteed not to cause name clashes and add it to `.cmake-format` for the proper spelling?